src/material/progress-bar/progress-bar.html

```diff
@@ -2,14 +2,14 @@
   The background div is named as such because it appears below the other divs and is not sized based
   on values.
 -->
-<svg width="100%" height="4" focusable="false" class="mat-progress-bar-background mat-progress-bar-element">
+<svg width="100%" height="4" focusable="false" class="mat-progress-bar-background mat-progress-bar-element" aria-hidden="true">
```

Comment:
Rather than repeating `aria-hidden`, it would probably be better to introduce an aria-hidden div around everything inside

Reply:
Good idea, added the div